synapse/media/_base.py

```diff
@@ -50,6 +50,24 @@
     "text/xml",
 ]
 
+# A list of all content types that are "safe" to be rendered inline in a browser.
+INLINE_CONTENT_TYPES = [
+    "text/css",
+    "text/plain",
+    "text/csv",
+    "application/json",
+    "application/ld+json",
+    # We allow images listed under the common image list
+    # https://developer.mozilla.org/en-US/docs/Web/Media/Formats/Image_types#common_image_file_types
+    # except SVGs
+    "image/apng",
+    "image/avif",
+    "image/gif",
+    "image/jpeg",
+    "image/png",
+    "image/webp",
+]
```

Comment:
@Half-Shot Is there a reason why video and audio content types aren't being included in this PR? :thinking:

Reply:
Seconding this comment. I was also wondering about this.

Reply:
Glowing Bear, for instance, has been showing inline video and audio (using native HTML widgets) for years.

Reply:
The list is now amended to follow what the matrix-react-sdk does.